anime-list-master.xml

```diff
@@ -6098,10 +6098,10 @@
   <anime anidbid="1742" tvdbid="164361" defaulttvdbseason="0" episodeoffset="" tmdbid="" imdbid="">
     <name>Choujuu Kishin Dancougar: Ushinawareta Mono-tachi e no Chinkonka</name>
   </anime>
-  <anime anidbid="1743" tvdbid="unknown" defaulttvdbseason="1" episodeoffset="" tmdbid="" imdbid="">
+  <anime anidbid="1743" tvdbid="345113" defaulttvdbseason="1" episodeoffset="" tmdbid="" imdbid="">
     <name>Super Doll Licca-chan</name>
   </anime>
-  <anime anidbid="1744" tvdbid="unknown" defaulttvdbseason="1" episodeoffset="" tmdbid="" imdbid="">
+  <anime anidbid="1744" tvdbid="73629" defaulttvdbseason="1" episodeoffset="" tmdbid="" imdbid="">
```

Comment:
Not Season 1

Reply:
It not the special like I originally though they dont seem to line up, so I'm removing this one.